src/components/QSnackBar/QSnackBar.vue

```diff
@@ -0,0 +1,235 @@
+<!-- eslint-disable vue/multi-word-component-names -->
+<template>
+  <div :class="{ 'q-animate-fadeIn': isOpen, 'q-animate-fadeOut': !isOpen}">
+    <div v-if="isOpen" :class="allClasses">
+      <q-row>
+        <q-col :md="10">
+          <p class="text-snackbar" :class="colorTextClass">{{ message }}</p>
+        </q-col>
+        <q-col :md="2">
+          <q-button
+              class="button-snackbar" variant="plain-rounded-full" :color="colorButton" size="small"
+              @click="Close">X
+          </q-button>
+        </q-col>
+      </q-row>
+    </div>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: 'QSnackBar',
+  props: {
+    variant: {
+      type: String,
+      default: "tile",
+      validator(value) {
+        const validVariant = ['tile', 'rounded', 'shaped', 'outline'];
+        return validVariant.includes(value);
+      },
+    },
+    position: {
+      type: String,
+      validator(value) {
+        const validPosition = ['top-center', 'bottom-center', 'top-right', 'top-left', 'bottom-left', 'bottom-right'];
+        return validPosition.includes(value);
+      },
+    },
+    open: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
Should be required.

Should be V-Model (https://vuejs.org/api/built-in-directives.html#v-model)